CHANGELOG.md

```diff
@@ -359,10 +361,6 @@ our [guidelines for writing a good changelog entry](https://github.com/biomejs/b
 
 #### Bug fixes
 
-- JavaScript lexer is now able to lex regular expression literals with escaped non-ascii chars ([#1941](https://github.com/biomejs/biome/issues/1941)).
-
-  Contributed by @Sec-ant
```

Comment:
Why was this removed?
Bad merge conflict resolution?

Reply:
Ah yes, that was indeed a wrongly merged conflict. I'll fix right away, thanks!

Reply:
https://github.com/biomejs/biome/pull/2321